### DISCUSSION TITLE
Batch favourite photos from album/collection view?

### DISCUSSION THREAD

**Discussion by nicoleritzer** [deckerst/aves#148] [Q&A] [answered]
Hello and thank you for the great gallery app, it's the only tool besides a full-on file explorer that can handle my frankly unwieldy collection of 10,000+ images.

Question: I have only been able to delete/move/etc files while in album view, and can only mark as favourite/"heart" them while viewing one picture at a time. Is it possible to mark multiple photos as favourite at the same time? 

I've taken a look at the Quick Actions in the Settings, but haven't seen the favourite button on the list in either Thumbnails or Viewer.

Thanks again!

**Comment by deckerst** [comment, marked as answer]
I'm glad the app is helpful to you.

Thanks for the suggestion, I'll tackle it in #149. You can subscribe to that issue to follow progress.
